packages/main/src/plugin/extension-loader.ts

```diff
@@ -787,7 +787,9 @@ export class ExtensionLoader {
         // eslint-disable-next-line @typescript-eslint/no-explicit-any
         thisArg?: any,
       ): containerDesktopAPI.Disposable {
-        return commandRegistry.registerCommand(command, callback, thisArg);
+        const registration = commandRegistry.registerCommand(command, callback, thisArg);
```

Comment:
We could have a utility method 

`return registerDisposable(commandRegistry.registerCommand,  disposables)`

so we get one line everywhere instead of three

```js
function registerDisposable(disposable, disposables): void {
    disposables.push(disposable)
    return disposable;
}
```